source/sd.c

```diff
@@ -66,22 +46,19 @@ int loadPrivLibs(void) {
     }
     else {
         LOG("Failed to load libkernel_sys.sprx");
-        return -4;
+        return -2;
     }
 
     return 0;
 }
 
 int generateSealedKey(uint8_t data[ENC_SEALEDKEY_LEN]) {
-    uint8_t dummy[0x30];
-    uint8_t sealedKey[ENC_SEALEDKEY_LEN];
+    uint8_t dummy[0x30]; UNUSED(dummy);
```

Comment:
any reason for these `dummy` arrays?

Reply:
Yes, they are there to prevent a crash. I think it's because of the kernel is killing the stack.